docs/tuning_fledge.rst

```diff
@@ -762,6 +767,113 @@ When collection is enabled the following counters will be collected for the stor
       - The size of the JSON payload in the delete calls to the storage layer for the given table.
       - There is little an end user can influence regarding the payload size, however it gives an indication of bandwidth usage for the storage API.
 
+Purge
+=====
+
+The purpose of the purge processes within Fledge is to control the usage of the storage system. Fledge has two different purge process that run, each of these purges a different aspect of the storage within the system;
```

Comment:
- different purge process -> different purge processes
- each of these purges -> each of which purges
- End with a period instead of a semi-colon